examples/js/shaders/BokehShader.js

```diff
@@ -75,7 +75,7 @@ THREE.BokehShader = {
 
 		'void main() {',
 
-		'	vec2 aspectcorrect = vec2( 1.0, aspect );',
+		'	vec2 aspectcorrect = vec2( 1.0 / aspect, 1.0 );',
```

Comment:
Can you please update the JSM file, too? Meaning `examples/jsm/shaders/BokehShader.js`.

Reply:
Sure, done.